Pyopenms tests still failing

```
...............................................................No source file annotated.
.F.................Error, intensity or m/z array is missing, skipping this spectrum
Error, intensity or m/z array is missing, skipping this spectrum
...................Skip mapping for chromatogram  since no precursor or product m/z was recorded.
.....................................sh: 1: gnuplot: not found
Warning (ModificationsDB::getModification): more than one modification with name 'Carbamidomethyl', residue '', specificity '5' found, picking the first one only.
Warning (ModificationsDB::getModification): more than one modification with name 'Oxidation', residue '', specificity '5' found, picking the first one only.
Calling 'gnuplot' on '_target_decoy' failed. Please create plots manually.
....................................................................................F......Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '143.004099999999994' is not uniquely defined by the given data. Using 'Ammonia-loss (N-term C)' to represent any of 'Ammonia-loss (N-term C), Ammonia-loss (N-term C), N-(L-isoaspartyl)-L-cysteine (N-term C)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '111.031999999999996' is not uniquely defined by the given data. Using 'Glu->pyro-Glu (N-term E)' to represent any of 'Glu->pyro-Glu (N-term E), Glu->pyro-Glu (N-term E), N-(L-isoglutamyl)-glycine (N-term E), O-(L-phenylalanyl)-L-serine (C-term)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Warning: For modification with mass 0.0, mass == massdiff. This is wrong. Please report it to the maintainer of the tool that wrote the pepXML. OpenMS will try to calculate it manually, assuming massdiff is correct.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 3 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '143.004099999999994' is not uniquely defined by the given data. Using 'Ammonia-loss (N-term C)' to represent any of 'Ammonia-loss (N-term C), Ammonia-loss (N-term C), N-(L-isoaspartyl)-L-cysteine (N-term C)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '111.031999999999996' is not uniquely defined by the given data. Using 'Glu->pyro-Glu (N-term E)' to represent any of 'Glu->pyro-Glu (N-term E), Glu->pyro-Glu (N-term E), N-(L-isoglutamyl)-glycine (N-term E), O-(L-phenylalanyl)-L-serine (C-term)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
.<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 3 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '143.004099999999994' is not uniquely defined by the given data. Using 'Ammonia-loss (N-term C)' to represent any of 'Ammonia-loss (N-term C), Ammonia-loss (N-term C), N-(L-isoaspartyl)-L-cysteine (N-term C)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 3 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '111.031999999999996' is not uniquely defined by the given data. Using 'Glu->pyro-Glu (N-term E)' to represent any of 'Glu->pyro-Glu (N-term E), Glu->pyro-Glu (N-term E), N-(L-isoglutamyl)-glycine (N-term E), O-(L-phenylalanyl)-L-serine (C-term)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Warning: For modification with mass 0.0, mass == massdiff. This is wrong. Please report it to the maintainer of the tool that wrote the pepXML. OpenMS will try to calculate it manually, assuming massdiff is correct.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 3 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '143.004099999999994' is not uniquely defined by the given data. Using 'Ammonia-loss (N-term C)' to represent any of 'Ammonia-loss (N-term C), Ammonia-loss (N-term C), N-(L-isoaspartyl)-L-cysteine (N-term C)'.
<Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.> occurred 2 times
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '111.031999999999996' is not uniquely defined by the given data. Using 'Glu->pyro-Glu (N-term E)' to represent any of 'Glu->pyro-Glu (N-term E), Glu->pyro-Glu (N-term E), N-(L-isoglutamyl)-glycine (N-term E), O-(L-phenylalanyl)-L-serine (C-term)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Modification '147.03540000000001' is not uniquely defined by the given data. Using 'Oxidation (M)' to represent any of 'Oxidation (M), Oxidation (M), oxidation to L-methionine sulfoxide (M), oxidation to L-methionine (R)-sulfoxide (M), oxidation to L-methionine (S)-sulfoxide (M)'.
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': Errors during parsing of aminoacid/terminal modification element:
Non-fatal error while loading '/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test.pep.xml': No modification description given. Trying to define by modification mass.
..........No source file annotated.
1 spectra and 0 chromatograms stored.
0 spectra and 0 chromatograms stored.
Attempting to call 'gnuplot' ...IsobaricNormalizer: max ratio deviation of alternative method is 0%
<1 spectra and 0 chromatograms stored.> occurred 4 times
IsobaricNormalizer: max ratio deviation of alternative method is 0%
Will analyse 2 peptides with a total of 5 transitions 
..................................................IMPORTED  /home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/pyopenms/__init__.py
run  testAASequence
Exception successfully triggered.
run  testAScore
run  testAdduct
run  testAttachment
run  testBase64
run  testCVMappingFile
run  testCVMappings
run  testCachedMzML
run  testChargePair
run  testChecksumType
run  testChromatogramPeak
run  testChromatogramToosl
run  testCoarseIsotopePatternGenerator
run  testCompNovoIdentification
run  _testParam
run  testCompNovoIdentificationCID
run  _testParam
run  testCompomer
run  testConfidenceScoring
run  testConsensusFeature
run  _testUniqueIdInterface
run  _testMetaInfoInterface
run  testConsensusIDAlgorithmAverage
run  testConsensusIDAlgorithmBest
run  testConsensusIDAlgorithmIdentity
run  testConsensusIDAlgorithmPEPIons
run  testConsensusIDAlgorithmPEPMatrix
run  testConsensusIDAlgorithmRanks
run  testConsensusIDAlgorithmSimilarity
run  testConsensusIDAlgorithmWorst
run  testConsensusMap
run  testConsensusMapNormalizerAlgorithmMedian
run  testConsensusMapNormalizerAlgorithmQuantile
run  testConsensusMapNormalizerAlgorithmThreshold
run  testConsensusXMLFile
run  testContactPerson
run  testControlledVocabulary
run  testConvexHull2D
run  testDPosition
run  testDTA2DFile
run  testDTAFile
run  testDataProcessing
run  _testMetaInfoInterface
run  testDataType
run  testDataValue
run  testDigestionEnzymeProtein
run  testDocumentIdentifier
run  testEDTAFile
run  testElement
run  testElementDB
run  testElutionPeakDetection
run  testEmpiricalFormula
run  testEnzymaticDigestionLogModel
run  testExperimentalDesign
run  testExperimentalSettings
run  testFASTAEntry
run  testFASTAFile
run  testFalseDiscoveryRate
run  _testParam
run  testFeature
run  _testMetaInfoInterface
run  _testUniqueIdInterface
run  testFeatureDeconvolution
run  _testParam
run  testFeatureFileOptions
run  testFeatureFinder
run  _testParam
run  testFeatureFinderAlgorithmIsotopeWavelet
run  testFeatureFinderAlgorithmPicked
run  testFeatureGrouping
run  testFeatureMap
run  _testUniqueIdInterface
run  testDataProcessing
run  _testMetaInfoInterface
run  testFeatureXMLFile
run  testFileDescription
run  testFileHandler
run  testFineIsotopePatternGenerator
run  testFloatDataArray
run  testGaussFitResult
run  testGaussFitter
run  testGradient
run  testHMMState
run  testHPLC
run  testHiddenMarkovModel
run  testIBSpectraFile
run  testIDDecoyProbability
run  testIDFilter
run  testIDMapper
run  testIDRipper
run  testIIMN
run  testIdXMLFile
run  testIdentification
run  _testMetaInfoInterface
run  testIdentificationHit
run  _testMetaInfoInterface
run  testIndexedMzMLDecoder
run  testIndexedMzMLFile
run  testInspectInfile
run  testInstrument
run  testInstrumentSettings
run  _testMetaInfoInterface
run  testIntegerDataArray
run  testInternalCalibration
run  testIonDetector
run  testIonSource
run  testIsotopeDistribution
<pyopenms.pyopenms_7.Peak1D object at 0x7feb6549c430>
<pyopenms.pyopenms_7.Peak1D object at 0x7feb6549c9f0>
run  testIsotopeMarker
run  testItraqConstants
run  testKernelMassTrace
run  testKroenikFile
run  testLinearResampler
run  _testParam
run  testLogType
run  testMRMAssay
run  testMRMDecoy
run  testMRMFeature
run  testMRMIonSeries
run  testMRMMapping
run  testMRMTransitionGroup
run  testMSChromatogram
run  _testMetaInfoInterface
run  testMSExperiment
run  _testMetaInfoInterface
run  testMSPFile
run  testMSQuantifications
run  testMSSpectrum
run  _testMetaInfoInterface
run  testSpectrumSetting
run  testMapAlignment
run  testMapAlignmentIdentification
run  testMapAlignmentTransformer
run  testMassAnalyzer
run  testMatrixDouble
run  testModificationDefinitionsSet
run  testModificationsDB
run  testMxxxFile
run  testMzIdentMLFile
run  testMzTab
run  testMzTabFile
run  testNASequence
run  testNumpressCoder
run  testNumpressConfig
run  testOptimizePeakDeconvolution
run  testParamXMLFile
run  testPeak
run  testPeakFileOptions
run  testPeakPickerHiRes
run  testPeakPickerMRM
run  testPeakTypeEstimator
run  testPepXMLFile
run  testPeptideAndProteinQuant
run  _testParam
run  testPeptideEvidence
run  testPeptideHit
run  _testMetaInfoInterface
run  testPeptideIdentification
run  _testMetaInfoInterface
run  testPeptideIndexing
run  testPeptideProteinResolution
run  testPercolatorOutfile
run  testPolarity
run  testPosteriorErrorProbabilityModel
run  _testParam
run  testPrecursor
run  testProcessingAction
run  testProduct
run  testProtXMLFile
run  testProteaseDB
run  testProteaseDigestion
run  testProteinHit
run  _testMetaInfoInterface
run  testProteinIdentification
run  _testMetaInfoInterface
run  testProteinResolver
run  testRNaseDB
run  testRNaseDigestion
run  testReactionMonitoringTransition
run  testResidue
run  testResidueDB
run  testRibonucleotide
run  testRibonucleotideDB
run  testRichPeak
run  _testMetaInfoInterface
run  _testUniqueIdInterface
run  testSample
run  testSeedListGenerator
run  testSemanticValidator
run  testSignalToNoiseEstimatorMedian
run  testSignalToNoiseEstimatorMedianChrom
run  testSoftware
run  testSourceFile
run  testSpectrumAlignment
run  testSpectrumIdentification
run  _testMetaInfoInterface
run  testSpectrumSetting
run  testString
b'blah'
b'blah'
blah
b'bl\xc3\xa4h'
b'bl\xc3\xa4h'
b'bl\xc3\xa4h'
b'bl\xc3\xa4h'
b'bl\xc3\xa4h'
run  testStringDataArray
run  testSvmTheoreticalSpectrumGeneratorTrainer
run  testSwathFile
run  testTOFCalibration
run  testTargetedExperiment
run  testTargetedExperimentHelper
run  testTransformationDescription
run  testTransformationModels
run  testTransformationXMLFile
run  testTransitionTSVFile
run  testType
run  testVersion
run  testXTandemInfile
run  testXTandemXMLFile
run  test_AcquisitionInfo
run  test_AnnotationState
run  test_BaseFeature
run  _testMetaInfoInterface
run  _testUniqueIdInterface
run  _testMetaInfoInterface
run  test_peptide_identifications_to_df
run b testAASequenceTutorial
DFPIANGER
DFPIANGERDFPIANGER
ANGER
Monoisotopic m/z of (M+2H)2+ is 509.7588010354211
Peptide DFPIANGER has molecular formula C44H67N13O15
===================================
Isotope 1017.7142391373 : 0.08042114973068237
Isotope 1018.7175939750999 : 0.042753059417009354
Isotope 1019.7209488128999 : 0.011149941012263298
Isotope 1020.7243036507 : 0.518894612789154
Isotope 1021.7276584885 : 0.2750795781612396
Isotope 1022.7310133263 : 0.07170164585113525
===================================
y3 ion : GER
y3 mz : 181.10268634957103
C13H24N6O6
C44H67N13O15
PEPTIDESEKUEM(Oxidation)CER
PEPTIDESEKUEM(Oxidation)CER
PEPTIDESEKUEMCER
PEPTIDESEKUEM(UniMod:35)CER
PEPTIDESEKUEM[147]CER
PEPTIDESEKUEM[147.03540001709996]CER
DFPIAM(Oxidation)GER
DFPIAM(Oxidation)GER
DFPIAM(Oxidation)GER
DFPIAM(Oxidation)GER
DFPIAM(Oxidation)GER
.(Dimethyl)DFPIAMGER True
DFPIAMGER.(Label:18O(2)) True
DFPIAMGER(Phospho) False
2
BSA MKWVTFISLLLLFSSAYSRGVFRRDTHKSEIAHRFKDLGE
ALB MKWVTFISLLFLFSSAYSRGVFRRDAHKSEVAHRFKDLGE
run b testDatastructuresTutorial
600.0 0.0
700.0 0.0
800.0 0.0
900.0 0.0
1000.0 0.0
1100.0 0.0
1200.0 0.0
1300.0 0.0
1400.0 0.0
1500.0 0.0
600.0 0.0
700.0 0.0
800.0 0.0
900.0 0.0
1000.0 0.0
1100.0 0.0
1200.0 0.0
1300.0 0.0
1400.0 0.0
1500.0 0.0
700.0 0.0
0.0 500.0 37.5
0.0 600.0 37.5
0.0 700.0 37.5
0.0 800.0 37.5
1.0 501.0 62.5
1.0 601.0 62.5
1.0 701.0 62.5
1.0 801.0 62.5
2.0 502.0 87.5
2.0 602.0 87.5
2.0 702.0 87.5
2.0 802.0 87.5
3.0 503.0 87.5
3.0 603.0 87.5
3.0 703.0 87.5
3.0 803.0 87.5
4.0 504.0 62.5
4.0 604.0 62.5
4.0 704.0 62.5
4.0 804.0 62.5
5.0 505.0 37.5
5.0 605.0 37.5
5.0 705.0 37.5
5.0 805.0 37.5
700.0
600.0 0.028565499931573868
700.0 0.1353352814912796
800.0 0.4111122786998749
900.0 0.8007373809814453
1000.0 1.0
1100.0 0.8007373809814453
1200.0 0.4111122786998749
1300.0 0.1353352814912796
1400.0 0.028565499931573868
1500.0 0.003865920240059495
700.0 0.1353352814912796
run b testDigestionTutorial
testDigestionTutorial
VASLR
VASLRETYGDMADCCEK
run b testElementTutorial
31.97207073 : 0.9492999911308289
32.971458 : 0.007600000128149986
33.967867 : 0.042899999767541885
35.967081 : 0.00019999999494757503
run b testEmpiricalFormulaTutorial
testEmpiricalFormulaTutorial
C1H6O2
{b'C': 1, b'H': 6, b'O': 2}
50.066950191400004 : 0.9886175990104675
51.0703050292 : 0.01137484423816204
52.073659867 : 7.574918527097907e-06
50.0 : 0.9886175990104675...
======================================================================
FAIL: @tests: FineIsotopePatternGenerator
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test000.py", line 33, in wrapper
    f(*a, **kw)
  File "/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test000.py", line 371, in testFineIsotopePatternGenerator
    assert len(iso_dist.getContainer()) == 56 # now 33 ?
AssertionError

======================================================================
FAIL: test_spectrum (unittests.test_DIAScoring.TestDIAScoring)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/ws/openms/PR/pyOMSDckr/OpenMS-build/pyOpenMS/tests/unittests/test_DIAScoring.py", line 64, in test_spectrum
    self.assertAlmostEqual (bseries_score, 1.0)
AssertionError: 0.0 != 1.0 within 7 places

----------------------------------------------------------------------
Ran 293 tests in 10.244s

FAILED (failures=2)

51.0 : 0.01137484423816204
52.0 : 7.574918527097907e-06
run b testResidueTutorial
run b testTheoreticalSpectrumGenTutorial
Spectrum 1 has 8 peaks.
Spectrum 2 has 30 peaks.
b'y1++' 88.07065719007102
b'b2++' 132.054955690071
b'y2++' 152.59195430172102
b'y1+' 175.134037913371
b'b3++' 180.58133780172102
b'y3++' 181.10268634957103
b'b4++' 237.12336997717102
b'y4++' 238.12415044527103
b'b2+' 263.10263491337105
b'b5++' 272.641927056921
b'y5++' 273.642707525021
b'y2+' 304.176632136671
b'b6++' 329.66339115262105
b'y6++' 330.184739700471
b'b7++' 358.17412320047106
b'b3+' 360.155399136671
b'y3+' 361.19809623237103
b'y7++' 378.711121812121
b'b8++' 422.6954203121211
b'y8++' 452.245328955671
b'b4+' 473.239463487571
b'y4+' 475.241024423771
b'b5+' 544.276577647071
b'y5+' 546.2781385832709
b'b6+' 658.3195058384711
b'y6+' 659.3622029341709
b'b7+' 715.3409699341711
b'y7+' 756.4149671574709
b'b8+' 844.383564157471
b'y8+' 903.4833814445709
<IsobaricNormalizer: max ratio deviation of alternative method is 0%> occurred 2 times
2 spectra and 0 chromatograms stored.
6 spectra and 0 chromatograms stored.
0 spectra and 1 chromatograms stored.
```

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -368,9 +368,9 @@ def testFineIsotopePatternGenerator():
     water = pyopenms.EmpiricalFormula("H2O")
     mw = methanol + water
     iso_dist = mw.getIsotopeDistribution(pyopenms.FineIsotopePatternGenerator(1e-20, False, False))
-    assert len(iso_dist.getContainer()) == 56 # now 33 ?
+    assert len(iso_dist.getContainer()) == 56
     iso_dist = mw.getIsotopeDistribution(pyopenms.FineIsotopePatternGenerator(1e-200, False, False))
-    assert len(iso_dist.getContainer()) == 88 # now 42
+    assert len(iso_dist.getContainer()) == 84
 
     c100 = pyopenms.EmpiricalFormula("C100")
     iso_dist = c100.getIsotopeDistribution(pyopenms.FineIsotopePatternGenerator(1e-200, False, False))

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -5270,7 +5270,7 @@ def testElementDB():
     assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()
     assert len(e2.getIsotopeDistribution().getContainer()) == 1
-    assert abs(e2.getIsotopeDistribution().getContainer()[0].getIntensity() - 0.1) < 1e-5
+    assert abs(e2.getIsotopeDistribution().getContainer()[0].getIntensity() - 1.0) < 1e-5
     # assert e == e2
 
     #  not yet implemented

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -5258,14 +5258,14 @@ def testElementDB():
 
     # changing existing elements in tests might have side effects so we define a new element
     # add first new element
-    e2 = edb.addElement(b"Kryptonite", b"@", 500, {999 : 0.7, 1000 : 0.3}, {999 : 999.99, 1000 : 1000.01}, False)
+    e2 = edb.addElement(b"Kryptonite", b"@", 500, {999 : 0.7, 1000 : 0.3}, {999 : 999.01, 1000 : 1000.01}, False)
     e2 = edb.getElement(pyopenms.String("@"))
     assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()
     assert len(e2.getIsotopeDistribution().getContainer()) == 2
     assert abs(e2.getIsotopeDistribution().getContainer()[1].getIntensity() - 0.3) < 1e-5
     # replace element
-    e2 = edb.addElement(b"Kryptonite", b"@", 500, {9999 : 0.9}, {9999 : 9999.9}, True)
+    e2 = edb.addElement(b"Kryptonite", b"@", 500, {9999 : 1.0}, {9999 : 9999.1}, True)
     e2 = edb.getElement(pyopenms.String("@"))
     assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -5256,14 +5256,21 @@ def testElementDB():
     e2 = edb.getElement(pyopenms.String("NE"))
     assert e2.getName() == "NewElement"
 
-    # replace oxygen
-    e2 = edb.addElement(b"Oxygen", b"O", 8, {16 : 0.7, 19 : 0.3}, {16 : 16.01, 19 : 19.01}, True)
-    e2 = edb.getElement(pyopenms.String("O"))
-    assert e2.getName() == "Oxygen"
+    # changing existing elements in tests might have side effects so we define a new element
+    # add first new element
+    e2 = edb.addElement(b"Cryptonite", b"@", 500, {999 : 0.7, 1000 : 0.3}, {999 : 999.99, 1000 : 1000.01}, False)
+    e2 = edb.getElement(pyopenms.String("@"))
+    assert e2.getName() == "Cryptonite"
     assert e2.getIsotopeDistribution()
     assert len(e2.getIsotopeDistribution().getContainer()) == 2
     assert abs(e2.getIsotopeDistribution().getContainer()[1].getIntensity() - 0.3) < 1e-5
-
+    # replace element
+    e2 = edb.addElement(b"Cryptonite", b"@", 500, {9999 : 0.9, 1000 : 0.1}, True)
+    e2 = edb.getElement(pyopenms.String("@"))
+    assert e2.getName() == "Cryptonite"
+    assert e2.getIsotopeDistribution()
+    assert len(e2.getIsotopeDistribution().getContainer()) == 1
+    assert abs(e2.getIsotopeDistribution().getContainer()[0].getIntensity() - 0.1) < 1e-5
     # assert e == e2
 
     #  not yet implemented

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -5265,7 +5265,7 @@ def testElementDB():
     assert len(e2.getIsotopeDistribution().getContainer()) == 2
     assert abs(e2.getIsotopeDistribution().getContainer()[1].getIntensity() - 0.3) < 1e-5
     # replace element
-    e2 = edb.addElement(b"Kryptonite", b"@", 500, {9999 : 0.9, 1000 : 0.1}, True)
+    e2 = edb.addElement(b"Kryptonite", b"@", 500, {9999 : 0.9}, {9999 : 9999.9}, True)
     e2 = edb.getElement(pyopenms.String("@"))
     assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()

--- a/src/pyOpenMS/tests/unittests/test000.py
+++ b/src/pyOpenMS/tests/unittests/test000.py
@@ -5258,16 +5258,16 @@ def testElementDB():
 
     # changing existing elements in tests might have side effects so we define a new element
     # add first new element
-    e2 = edb.addElement(b"Cryptonite", b"@", 500, {999 : 0.7, 1000 : 0.3}, {999 : 999.99, 1000 : 1000.01}, False)
+    e2 = edb.addElement(b"Kryptonite", b"@", 500, {999 : 0.7, 1000 : 0.3}, {999 : 999.99, 1000 : 1000.01}, False)
     e2 = edb.getElement(pyopenms.String("@"))
-    assert e2.getName() == "Cryptonite"
+    assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()
     assert len(e2.getIsotopeDistribution().getContainer()) == 2
     assert abs(e2.getIsotopeDistribution().getContainer()[1].getIntensity() - 0.3) < 1e-5
     # replace element
-    e2 = edb.addElement(b"Cryptonite", b"@", 500, {9999 : 0.9, 1000 : 0.1}, True)
+    e2 = edb.addElement(b"Kryptonite", b"@", 500, {9999 : 0.9, 1000 : 0.1}, True)
     e2 = edb.getElement(pyopenms.String("@"))
-    assert e2.getName() == "Cryptonite"
+    assert e2.getName() == "Kryptonite"
     assert e2.getIsotopeDistribution()
     assert len(e2.getIsotopeDistribution().getContainer()) == 1
     assert abs(e2.getIsotopeDistribution().getContainer()[0].getIntensity() - 0.1) < 1e-5

